syntax = "proto3";

package user;

service UserService {
    rpc GetUser (UserRequest) returns (UserResponse) {}
}

message UserRequest {
    int32 user_id = 1;
}

message UserResponse {
    int32 user_id = 1;
    string name = 2;
    // string email = 3;  // Campo eliminado
    reserved 3;  // Reservamos el número de campo para evitar su reutilización accidental
}